sample-input/sph-factors/assembly/sph-region-avg.py

```diff
@@ -0,0 +1,119 @@
+import matplotlib.pyplot as plt
```

Comment:
I actually haven't run this problem on a cluster, but I have seen this error before (when debugging an earlier implementation of the `openmoc.plotter` module for AWS in 2013). In particular, we avoid this issue with Matplotlib in `openmoc.plotter` with [`matplotlib.use('Agg')`](https://github.com/mit-crpg/OpenMOC/blob/develop/openmoc/plotter.py#L18) and [`plt.ioff()`](https://github.com/mit-crpg/OpenMOC/blob/develop/openmoc/plotter.py#L18) here.

Since this is one (the only?) example input file which uses Matplotlib directly rather than via `openmoc.plotter`, I think all I need to do is move my `matplotlib` import after the `openmoc.plotter` import. I'll try that and see if it works.